webapp/channels/src/packages/mattermost-redux/src/constants/telemetry.ts

```diff
@@ -6,13 +6,23 @@ export const TrackProfessionalSKU = 'professional';
 export const TrackEnterpriseSKU = 'enterprise';
 
 // Features
-export const TrackGroupsFeature: string = 'custom_groups';
+export const TrackGroupsFeature = 'custom_groups';
+export const TrackPassiveKeywordsFeature = 'passive_keywords';
 
 // Events
-export const TrackInviteGroupEvent: string = 'invite_group_to_channel';
+export const TrackInviteGroupEvent = 'invite_group_to_channel';
+export const TrackPassiveKeywordsEvent = 'add_or_update_passive_keywords';
```

Comment:
1/5  Should add and update considered as two different user actions? The code indicates there's no difference between add/update, so perhaps the event name could be renamed to `update_passive_keywords`. Otherwise it might make more sense to break it in two events (`add_passive_keywords` or `update_passive_keywords`).

Reply:
changed to update